service/controller/v2/credential/credential.go

```diff
@@ -0,0 +1,82 @@
+package credential
+
+import (
+	"github.com/giantswarm/microerror"
+	"k8s.io/api/core/v1"
+	apismetav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+
+	"github.com/giantswarm/azure-operator/client"
+	"github.com/giantswarm/azure-operator/service/controller/v2/key"
+)
+
+const (
+	ClientIDKey       = "clientID"
+	ClientSecretKey   = "clientSecret"
+	SubscriptionIDKey = "subscriptionID"
+	TenantIDKey       = "tenantID"
```

Comment:
Shouldn't they be `azure.azureoperator.clientid`, etc. to match with what we have in aws-operator?